hs/src/Reach/Connector/ALGO.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE CPP #-}
```

Comment:
I am anxious about how annoying CPP is. Idea: Create `Reach.Connector.ALGO_Verify` and have it use the macro and have a dummy implementation. This way we are also type-checking the code even when there's no Everest?

Reply:
Do you mean
1. `Reach.Connector.ALGO`  has no `CPP`;
2. `Reach.Connector.ALGO` always runs `Reach.Connector.ALGO_Verify.run`
3. if Everest is around, `Reach.Connector.ALGO_Verify.run = Reach.Closed.TAF.Verify.run`; and
4. otherwise, `Reach.Connector.ALGO_Verify.run _ _ = return ()`?

Reply:
Exactly